src/world.mojo

```diff
@@ -507,7 +508,7 @@ struct World:
     @always_inline
     fn is_locked(self) -> Bool:
         """
-        Rreturns whether the world is locked by any queries.
+        Returns whether the world is locked by any queries.
```

Comment:
I would probably raise a `NotImplementedError` here. But you probably already use it somewhere... You should definitely implement this before starting queries.

Reply:
I will implement the lock prior to queries. If I raise an error here, though, testing will not be possible.